adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -156,15 +156,15 @@ private[adam] class FastqRecordConverter extends Serializable with Logging {
                                               qual: String,
                                               readInFragment: Int,
                                               readPaired: Boolean = true,
-                                              recordGroupOpt: Option[String] = None): AlignmentRecord = {
+                                              readGroupOpt: Option[String] = None): AlignmentRecord = {
```

Comment:
While we're refactoring, might be nice to also go with consistent `opt` prefix on Options? --> `optReadGroup` instead of `readGroupOpt`

Reply:
+1